dspace-api/src/main/java/org/dspace/authority/orcid/Orcidv2AuthorityValue.java

```diff
@@ -1,57 +1,39 @@
-/**
- * The contents of this file are subject to the license and copyright
- * detailed in the LICENSE and NOTICE files at the root of the source
- * tree and available online at
- *
- * http://www.dspace.org/license/
- */
 package org.dspace.authority.orcid;
 
-import org.dspace.authority.AuthorityValue;
-import org.dspace.authority.AuthorityValueServiceImpl;
-import org.dspace.authority.PersonAuthorityValue;
-import org.dspace.authority.orcid.model.Bio;
-import org.dspace.authority.orcid.model.BioExternalIdentifier;
-import org.dspace.authority.orcid.model.BioName;
-import org.dspace.authority.orcid.model.BioResearcherUrl;
 import org.apache.commons.lang.StringUtils;
-import org.apache.log4j.Logger;
 import org.apache.solr.common.SolrDocument;
 import org.apache.solr.common.SolrInputDocument;
+import org.dspace.authority.AuthorityValue;
+import org.dspace.authority.AuthorityValueServiceImpl;
+import org.dspace.authority.PersonAuthorityValue;
+import org.dspace.utils.DSpace;
+import org.orcid.jaxb.model.record_v2.*;
 
 import java.util.*;
 
 /**
- *
- * @author Antoine Snyers (antoine at atmire.com)
- * @author Kevin Van de Velde (kevin at atmire dot com)
- * @author Ben Bosman (ben at atmire dot com)
- * @author Mark Diggory (markd at atmire dot com)
+ * Created by jonas - jonas@atmire.com on 12/04/2018.
  */
-public class OrcidAuthorityValue extends PersonAuthorityValue {
+public class Orcidv2AuthorityValue extends PersonAuthorityValue{
```

Comment:
Same here, needs code comments.

Reply:
We will include some basic code comments. They were not present in the ORCID v1 API, and therefore not included here yet